metabase_api/metabase_api.py

```diff
@@ -642,6 +642,35 @@ def create_segment(self, segment_name, column_name, column_values, segment_descr
 
 
 
+    def create_dashboard(self, name, description=None, parameters=None, cache_ttl=None, collection_id=None, collection_position=None, collection_name=None):
```

Comment:
"parameters" and "cache_ttl" are not used in the function.  
The function should allow passing custom json as argument to customize the dashboard (I assume the "parameters" option was supposed to do that). Currently only the name and description of the dashboard are customizable using this function.

Reply:
Got it, sorry to bother, I'm correcting this asap